easybuild/easyconfigs/a/AFNI/AFNI-linux_openmp_64-goolf-1.5.14.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+
+
+name = 'AFNI'
+version = 'linux_openmp_64'
+# see VersionDate in caps3 output
+
+easyblock = 'PackedBinary'
+
+homepage = 'http://afni.nimh.nih.gov'
+description = """Free software for analysis and display of FMRI data """
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+source_urls = [('http://afni.nimh.nih.gov/pub/dist/tgz/')]
+sources = ['%(version)s.tgz']
+
+dependencies = [
+    ('R', '3.1.2'),
+    ('PyQt', 'x11-gpl-4.11.3'),
+]
+
+osdependencies = [
+    ('libXp','tcsh')
+]
```

Comment:
we have both of these supported in EasyBuild, so I'd prefer to not included these as OS dependencies...

the `libXp` is a can of worms though, since it pulls in quite a bit of other stuff...